iocore/hostdb/HostDB.cc

```diff
@@ -609,8 +609,8 @@ HostDBProcessor::getby(Continuation *cont, const char *hostname, int len, sockad
                        HostResStyle host_res_style, int dns_lookup_timeout)
 {
   HostDBHash hash;
-  EThread *thread   = this_ethread();
-  ProxyMutex *mutex = thread->mutex.get();
+  EThread *thread       = this_ethread();
+  Ptr<ProxyMutex> mutex = thread->mutex;
```

Comment:
You should be able to use `auto` here, e.g.
```
auto mutex = thread->mutex;
```
I will have to defer to @zwoop if that's stylistically preferred.